label_studio/io_storages/all_api.py

```diff
@@ -108,6 +114,8 @@ def list(self, request, *args, **kwargs):
     name='get',
     decorator=swagger_auto_schema(
         tags=['Storage'],
+        x_fern_sdk_group_name='export_storage',
+        x_fern_sdk_method_name='list',
```

Comment:
and here? https://github.com/HumanSignal/label-studio/pull/5925/files?diff=unified&w=0#r1628559192

Reply:
same as https://github.com/HumanSignal/label-studio/pull/5925#discussion_r1629201461